cli.js

```diff
@@ -99,6 +110,7 @@ const USAGE_INFO = `
   ${GET_ENCODED_INIT_ARGS_CMD}        ❍ Calculate the initializer function arguments in ABI encoded format.
   ${SHOW_EXISTING_CONTRACTS_CMD} ❍ Show list of existing logic contract addresses on various blockchains.
   ${ADD_SUPPORTED_TOKEN_CMD}      ❍ Adds token at ${ETH_ADDRESS_ARG} to the supported tokens in vault at ${DEPLOYED_ADDRESS_ARG}.
+  ${SET_WETH_ADDRESS_CMD}       ❍ Sets the WETH Unwrapper contract at ${ETH_ADDRESS_ARG} to the supported tokens in vault at ${DEPLOYED_ADDRESS_ARG}.
```

Comment:
Description is confusing, since the supported tokens aren't relevant here at all. 

```suggestion
  ${SET_WETH_ADDRESS_CMD}       ❍ Sets the WETH unwrapper contract at ${ETH_ADDRESS_ARG} in the vault at ${DEPLOYED_ADDRESS_ARG}.
```